hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -1561,6 +1561,10 @@ public boolean isExecutorMetricsEnabled() {
         getStringOrDefault(HoodieMetricsConfig.EXECUTOR_METRICS_ENABLE, "false"));
   }
 
+  public boolean isMetricsCommonPrefixEnabled() {
```

Comment:
Why do you need another config. Check this PR: https://github.com/apache/hudi/pull/4274/files. If you want remove the infer function for METRICS_REPORTER_PREFIX, so it does not use the table name as default and the user can set a prefix using METRICS_REPORTER_PREFIX. Does it make sense?

Reply:
Ohh, I didnt notice while rebasing changes. I think I can use #4274. I will close this PR.

Reply:
No problem. Thanks @t0il3ts0ap